src/main/java/core/basesyntax/operation/impl/OperationServicePurchaseImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.operation.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationService;
+
+public class OperationServicePurchaseImpl implements OperationService {
```

Comment:
it's better to create separate classes for each operation instead of having only two - for addition and subtraction, cause it will make your code maintainable for future changes - fro example if the client changes the requirements and two addition operations will have different logic

Reply:
okay, but they will be identical

Reply:
for this situation you're right, but it's better to make such decision here even if we won't escape from code duplication